src/editor/contentState/selectionCtrl.js

```diff
@@ -0,0 +1,17 @@
+const selectionCtrl = ContentState => {
+  ContentState.prototype.tableCellHandler = function (event) {
+    const { start } = this.cursor
+    const block = this.getBlock(start.key)
```

Comment:
Please use `deconstruction` syntax.

```javascript
const { type, text, key } = block
```

this will make codes shorter and more readable.

Reply:
If start block is not the end block, and we should also consider this.

\|\<start cursor\>hello|nice|
\|---   | --- |
\|\<end cursor\>| foo bar|

I think if start.key is not equal end.key, we should select all the table content? if end block is not a table cell, we should select all the document? But I am not very sure, what's your opinions?